new_features.txt

```diff
@@ -0,0 +1,4 @@
+We add new features for iur site:
+1. login and password
+2. main photo (car photo)
+3. Add name of our company in the centre of the cite page
```

Comment:
Add new line please

Reply:
@Nellyougo I added new line! Please check it! %)

Reply:
@Nellyougo hahaha